package.json

```diff
@@ -79,8 +81,8 @@
     "lint": "tslint --format verbose --project . --exclude \"{build,node_modules}/**\" --exclude \"**/*.d.ts\" \"**/*.ts\"",
     "lint:fix": "npm run lint -- --fix",
     "start": "atm-start",
-    "test": "mocha --require espower-typescript/guess \"test/**/*Test.ts\"",
-    "test:one": "mocha --require espower-typescript/guess \"test/**/${TEST:-*Test.ts}\"",
+    "test": "mocha --timeout 10000 --require espower-typescript/guess \"test/**/*Test.ts\"",
```

Comment:
I had to extend the timeout, because the default 2000ms is not always enough for the java license generator to complete and this was causing intermittent test failures. 10000ms is very excessive, but I'm not sure what would be the execution times no other machines.

Reply:
I'll check it out CircleCI and see how long it takes there as well

Reply:
On my machine it was between ~1900ms - ~2400ms so sometimes it would pass even with defaults but more often at least one of the tests would fail. Depending on CircleCI time we can adjust the timeout setting

Reply:
:+1: